common-app-lib/src/cmr/common_app/services/search/parameter_validation.clj

```diff
@@ -229,7 +260,7 @@
 
 (def standard-valid-params
   "The set of standard valid query level parameters."
-  #{:page-size :page-num :offset :sort-key :result-format :options})
+  #{:offset :options :page-num :page-size :result-format :scroll :scroll-id :sort-key})
```

Comment:
No, I'm just stupid and got confused. It should be a header, not a parameter. She `scroll` flag is a parameter.